docs/languages/javascript.md

```diff
@@ -213,6 +213,58 @@ Organize imports can also be automatically when you save a JavaScript file by se
     "source.organizeImports": true
 }
 ```
+## Code Actions on Save
+
+The  `editor.codeActionsOnSave` setting lets you configure a set of Code Actions that are run when a file is saved. For example, you can enable organize imports on save by setting:
```

Comment:
The content looks good but let's try to tighten it up. These documents quick overviews of features available for js/ts so we try to keep each section fairly small.

I’d say keep the first example and the list of available code actions. You can link to documentation on language specific settings and other more advanced features